packages/web/docs/src/app/gateway/federation-compatible-benchmarks/index.tsx

```
@@ -90,26 +89,22 @@
     <div className="mt-6 flex flex-wrap gap-2 whitespace-nowrap text-xs text-green-800 sm:gap-4">
       <div className="flex gap-2 max-sm:-mx-1 max-sm:w-full sm:contents">
         <div className="flex items-center gap-1">
-          <CheckmarkIcon className="size-4" style={{ color: functionalTones.positiveDark }} />{' '}
-          Passed tests
+          <CheckmarkIcon className="size-4" className="text-positive-dark" /> Passed tests
         </div>
         <div className="flex items-center gap-1">
-          <XIcon className="size-4" style={{ color: functionalTones.criticalDark }} /> Failed tests
+          <XIcon className="size-4" className="text-critical-dark" /> Failed tests
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)